src/core/Akka.Remote/Configuration/Remote.conf

```diff
@@ -43,7 +43,7 @@ akka {
       "Akka.Routing.ScatterGatherFirstCompletedPool, Akka" = akka-misc
       "Akka.Routing.TailChoppingPool, Akka" = akka-misc
       "Akka.Routing.ConsistentHashingPool, Akka" = akka-misc
-      "Akka.Configuration.Config, Akka" = akka-misc
+      "Hocon.Config, Hocon.Configuration" = akka-misc
```

Comment:
So this represents a breaking change to the wire format, which will disrupt anyone who attempts to upgrade a live, running cluster from 1.3* to 1.4*.

We might be able to solve that without too much headache. We would need to add a little adapter here to convert between the `Akka.Config` and `Hocon.Config` type and register both as part of the `akka-misc` serializer. I have some ideas on how we can do that.